### PR TITLE
DISTPG-539 Release notes 15.2

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	modified:   docs/index.md
	modified:   docs/release-notes.md
	modified:   mkdocs-base.yml
new doc: docs/release-notes-v15.2.md

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ PostgreSQL
 
 * [HAProxy](http://www.haproxy.org/) - a high-availability and load-balancing solution 
 
+* [`pgpool`](https://git.postgresql.org/gitweb/?p=pgpool2.git;a=summary) - a middleware between PostgreSQL server and client for high availability, connection pooling and load balancing.
+
+* [PostGIS](https://trac.osgeo.org/postgis/) - a spatial database extender for PostgreSQL. It allows GIS (Geographic Information Systems) objects to be stored in the database.
```
Comment (ImTheKai) [inline docs/index.md]: We don't ship PostGIS with this update

File: docs/release-notes-v15.2.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+|[PgBouncer](https://www.pgbouncer.org/)          |1.18.0    | a lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.8   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)|2.0.0 | collects and aggregates statistics for PostgreSQL and provides histogram information.|
+| [PostgreSQL Common](https://salsa.debian.org/postgresql/postgresql-common)| 247 | PostgreSQL database-cluster manager. It provides a structure under which multiple versions of PostgreSQL may be installed and/or multiple clusters maintained at one time.|
+|[wal2json](https://github.com/eulerto/wal2json)  |2.5       | a PostgreSQL logical decoding JSON output plugin|
+|[HAProxy](http://www.haproxy.org/) | 2.5.11 | a high-availability and load-balancing solution |
+| [pgpool2](https://git.postgresql.org/gitweb/?p=pgpool2.git;a=summary) | 4.4.2 | a middleware between PostgreSQL server and client for high availability, connection pooling and load balancing.|
+| [PostGIS](https://trac.osgeo.org/postgis/) |  | a spatial database extender for PostgreSQL.|
```
Comment (ImTheKai) [inline docs/release-notes-v15.2.md]: We don't ship PostGIS in this release

**Review by ImTheKai — CHANGES_REQUESTED** [review]

File: docs/release-notes-v15.2.md (hunk 2 above)
Comment (nastena1606) [reply docs/release-notes-v15.2.md]: Updated. Thanks for spotting it!

File: docs/index.md (hunk 1 above)
Comment (nastena1606) [reply docs/index.md]: Updated

**Review by ImTheKai — APPROVED** [review]

**Review by Naeem-Akhter — APPROVED** [review]
LGTM!

**Pull request merged**